test/setup/main.tf

```diff
@@ -13,6 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+locals {
+  per_module_services = {
+    simple-secret = [
+      "secretmanager.googleapis.com"
```

Comment:
shouldn't this also include,

* `pubsub.googleapis.com`:  Use Pub/Sub for notifications using topics field
* `cloudkms.googleapis.com`: using cmek for encryption using customer_managed_encryption field